app/models/license.rb

```diff
@@ -14,21 +14,10 @@ class License < ApplicationRecord
   include Diffable
 
   belongs_to :policy
-
-  # FIXME(ezekg) The exists? check is for backwards compatiblity. Once we
-  #              fully denormalize product_id for all licenses, this will
-  #              turn into a belongs_to automatically. We can safely
-  #              remove this once that happens.
-  #
   # NOTE(ezekg) This is a denormalized association and is automatically
   #             pulled in from the policy. Purposefully defined after
   #             the policy association for order of validation.
-  if exists?(product_id: nil)
-    has_one :product, through: :policy
-  else
-    belongs_to :product
```

Comment:
Since we're using regular Rails migrations now for data migrations, this is no longer needed since the new application code won't be released until the data migration is completed.